python/shotgun_model/simple_shotgun_hierarchy_model.py

```diff
@@ -31,7 +31,7 @@ class SimpleShotgunHierarchyModel(ShotgunHierarchyModel):
     can also be subclassed from this class.
     """
 
-    def load_data(self, seed_entity_field, path=None, entity_fields=None):
+    def load_data(self, seed_entity_field, root=None, entity_fields=None):
```

Comment:
That's cool. Perhaps we should add back a `path=None` and have a more direct `raise` with a deprecation error so that it's clear to people when they try to use `path`? We could add a ticket to remove it after some time as well. Not a huge deal, but maybe better than the standard error saying unrecognized argument or whatever.